helper/nmt.go

```diff
@@ -0,0 +1,32 @@
+package helper
+
+import (
+	"os"
+	"strings"
+
+	"github.com/paketo-buildpacks/libpak/bard"
+)
+
+type NMT struct {
+	Logger bard.Logger
+}
+
+func (n NMT) Execute() (map[string]string, error) {
+
+	if s, ok := os.LookupEnv("BPL_JAVA_NMT_ENABLED"); ok && strings.ToLower(s) == "false" {
+		n.Logger.Info("Disabling Java Native Memory Tracking")
+		return nil, nil
+	}
+
+	n.Logger.Info("Enabling Java Native Memory Tracking")
+	var values []string
+	if s, ok := os.LookupEnv("JAVA_TOOL_OPTIONS"); ok {
+		values = append(values, s)
+	}
+	values = append(values, "-XX:+UnlockDiagnosticVMOptions", "-XX:NativeMemoryTracking=summary", "-XX:+PrintNMTStatistics")
```

Comment:
I think that the `-XX:NativeMemoryTracking` option should be configurable. We should add another env variable, like `BPL_JAVA_NMT_TRACKING` and default it to `summary`. That would allow for the user to enable `detail` level if necessary.

Reply:
Made this change - I used BPL_JAVA_NMT_LEVEL rather than TRACKING as it fits the value purpose and matches the JDK docs description of this flag's value